services/ui-src/src/components/forms/Form.tsx

```diff
@@ -36,9 +36,12 @@ export const Form = ({ id, formJson, onSubmit, children, ...props }: Props) => {
   };
 
   // hydrate and create form fields using formFieldFactory
-  const formFieldsToRender = (fields: FormField[]) => {
-    const hydratedFields = hydrateFormFields(fields, reportData);
-    return formFieldFactory(hydratedFields);
+  const renderFormFields = (fields: FormField[]) => {
+    let fieldsToRender = fields;
+    if (reportData) {
+      fieldsToRender = hydrateFormFields(fields, reportData);
+    }
+    return formFieldFactory(fieldsToRender);
```

Comment:
To fix the bug Garrett pointed out where two reports and hydrate the others fields, you could always do something like this:

```suggestion
    const fieldsToRender = reportData
      ? hydrateFormFields(fields, reportData)
      : hydrateFormFields(fields, {});
    return formFieldFactory(fieldsToRender);
```